### PR TITLE
Features/wordcloud

### PR DESCRIPTION
**Pull request by AnalystSubranjit**
1.  Updated the SIdebar components
2.  Added  Quick demo section
3. Added upload file widget
4. Bug fix related to wordcloud function call 
5. Minor tweaks

### CODE REVIEW

**Comment by aquibjaved** [comment]
Screenshot 2021-06-21 at 8 33 06 AM
@AnalystSubranjit, somthing is missing here

File: app.py [hunk 1]
```diff
@@ -9,8 +9,12 @@
 )
 
 
-docx_file = st.file_uploader("Load Sample File",type=['txt'])#,'docx','pdf'])
-# if docx_file:
+docx_file = st.file_uploader("Load Sample File",type=['txt','csv'])#,'docx','pdf'])
+#TODO: add functionality for file upload
+if docx_file:
+	st.write("This functionality is under development")
```
Comment (aquibjaved) [inline app.py:15]: okay got it

**Comment by AnalystSubranjit** [comment]
@aquibjaved check now

**Review by aquibjaved — CHANGES_REQUESTED** [review]
I think we should first implement this, 

- [ ] Uploading a data source (could be csv, tsv..etc files)
- [ ] Then providing an option in UI to select the column on which we want to the analysis
- [ ] wordcloud

File: app.py (hunk 1 above)
Comment (AnalystSubranjit) [reply app.py:15]: Yeah will these functionalities in the next commit

**Comment by AnalystSubranjit** [comment]
Done!

**Review by aquibjaved — CHANGES_REQUESTED** [review]

**Pull request merged**